partEZ/app/Http/Controllers/EventController.php

```diff
@@ -254,23 +252,10 @@ public function validatePoll( $eid )
 
     }
 
-    public function getVotes($pid, $oid)
-    {//TODO: what is this counting? Not the votes, but the options?
```

Comment:
Currently this function isn't being called anywhere. It's been moved to the poll options model so we can find it there when we need it.